<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>persian</class>
 <widget class="QMainWindow" name="persian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bestsync</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="6" column="1" colspan="10">
       <layout class="QHBoxLayout" name="horizontalLayout_97">
        <item>
         <widget class="QTreeView" name="treeView_7"/>
        </item>
       </layout>
      </item>
      <item row="2" column="13">
       <layout class="QHBoxLayout" name="horizontalLayout_91">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_67">
          <property name="toolTip">
           <string>جا به جایی</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../swap.png</normaloff>../swap.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="14">
       <layout class="QHBoxLayout" name="horizontalLayout_94">
        <item>
         <widget class="QComboBox" name="comboBox_26"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_39">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="11">
       <layout class="QHBoxLayout" name="horizontalLayout_92">
        <item>
         <widget class="QToolButton" name="button_7">
          <property name="toolTip">
           <string>اضافه کردن پوشه</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../add.png</normaloff>../add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_68">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../minus.png</normaloff>../minus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_25"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_37">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="11">
       <layout class="QHBoxLayout" name="horizontalLayout_93">
        <item>
         <widget class="QPushButton" name="pushButton_38">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string extracomment="File time &amp; size">مقایسه</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../file.png</normaloff>../file.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="14">
       <layout class="QHBoxLayout" name="horizontalLayout_95">
        <item>
         <widget class="QPushButton" name="pushButton_40">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>همسان سازی</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../sync.png</normaloff>../sync.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="13">
       <layout class="QHBoxLayout" name="horizontalLayout_96">
        <item>
         <widget class="QToolButton" name="toolButton_69">
          <property name="toolTip">
           <string>تنظیمات مقایسه</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_70">
          <property name="toolTip">
           <string>فیلتر</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_71">
          <property name="toolTip">
           <string>تنظیمات یکسان سازی</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../settings.png</normaloff>../settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="14" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_103">
        <item>
         <widget class="QTableView" name="tableView_14">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="13" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_104">
        <item>
         <widget class="QListWidget" name="listWidget_13"/>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_14"/>
        </item>
       </layout>
      </item>
      <item row="3" column="13">
       <layout class="QHBoxLayout" name="horizontalLayout_105">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>مبدا</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>مقصد</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="7" colspan="4">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>مرور کلی</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="10">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>آخرین یکسان سازی</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="toolButton_75">
        <property name="toolTip">
         <string>ذخیره به عنوان</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../logo (2).png</normaloff>../logo (2).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="14">
       <layout class="QHBoxLayout" name="horizontalLayout_101">
        <item>
         <widget class="QComboBox" name="comboBox_28"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_42">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="toolButton_76">
        <property name="toolTip">
         <string>Save as batch job</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../terminal.png</normaloff>../terminal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButton_77">
        <property name="toolTip">
         <string>ذخیره</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../save.png</normaloff>../save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="11" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_102">
        <item>
         <widget class="QTableView" name="tableView_13">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_72">
        <property name="toolTip">
         <string>جدید</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../new.png</normaloff>../new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout_98"/>
      </item>
      <item row="3" column="11">
       <layout class="QHBoxLayout" name="horizontalLayout_100">
        <item>
         <widget class="QToolButton" name="toolButton_73">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../minus.png</normaloff>../minus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_27"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_41">
          <property name="text">
           <string>جستجو</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButton_74">
        <property name="toolTip">
         <string>باز کردن</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../folder.png</normaloff>../folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="3" colspan="10">
       <layout class="QHBoxLayout" name="horizontalLayout_99">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget_7">
          <property name="gridVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>پوشه</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="actionSave_as_batch_job_2"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>عملیات</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>ابزار</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>زبان</string>
     </property>
     <addaction name="actionEnglish_2"/>
    </widget>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>کمک</string>
    </property>
    <addaction name="action_22"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>../new.png</normaloff>../new.png</iconset>
   </property>
   <property name="text">
    <string>جدید</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset>
     <normaloff>../open-file-button.png</normaloff>../open-file-button.png</iconset>
   </property>
   <property name="text">
    <string>باز کردن</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset>
     <normaloff>../save.png</normaloff>../save.png</iconset>
   </property>
   <property name="text">
    <string>ذخیره</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>ذخیره به عنوان</string>
   </property>
  </action>
  <action name="actionSave_as_batch_job_2">
   <property name="text">
    <string>Save as batch job</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset>
     <normaloff>../file.png</normaloff>../file.png</iconset>
   </property>
   <property name="text">
    <string>شروع مقایسه</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset>
     <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات مقایسه</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset>
     <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات فیلتر</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset>
     <normaloff>../settings.png</normaloff>../settings.png</iconset>
   </property>
   <property name="text">
    <string>تنظیمات یکسان سازی</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_17">
   <property name="icon">
    <iconset>
     <normaloff>../sync.png</normaloff>../sync.png</iconset>
   </property>
   <property name="text">
    <string>شروع یکسان سازی</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset>
     <normaloff>../binoculars.png</normaloff>../binoculars.png</iconset>
   </property>
   <property name="text">
    <string>یافتن...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>صدور لیست</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="icon">
    <iconset>
     <normaloff>../united-kingdom.png</normaloff>../united-kingdom.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_22">
   <property name="icon">
    <iconset>
     <normaloff>../faq.png</normaloff>../faq.png</iconset>
   </property>
   <property name="text">
    <string>مشاهده کمک</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
